starter-code/css/main.css

```diff
@@ -62,137 +83,137 @@ nav a {
 .main h1 {
   font-size: 60px;
   font-weight: bold;
-  color:#ffffff;
+  color: #ffffff;
 }
 
 .main p {
   color: #ffffff;
   font-size: 30px;
 }
 
-.background{
-    background-color: #ffffff;
-    background-size: cover;
-    background-repeat: no-repeat;
+.background {
+  background-color: #ffffff;
+  background-size: cover;
+  background-repeat: no-repeat;
 }
 
-h2{
-    font-size: 40px;
-    font-weight: bold;
-    color: #1a1a1a;
-    text-align: center
+h2 {
+  font-size: 40px;
+  font-weight: bold;
+  color: #1a1a1a;
+  text-align: center;
 }
 
-.container{
-    width: 100%;
-    margin: 0 auto;
-    padding: 4% 0;
-    display: flex;
-    flex-direction: row;
-    justify-content: space-around;
+.container {
+  width: 100%;
+  margin: 0 auto;
+  padding: 4% 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
 }
 
-.section-1{
-    display:flex;
-    justify-content: center;
-    align-items: center;
-    flex-direction: column;
+.section-1 {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
 }
 
-.section-2{
-    display:flex;
-    justify-content: center;
-    align-items: center;
-    flex-direction: column;
+.section-2 {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
 }
 
-.section-3{
-    display:flex;
-    justify-content: center;
-    align-items: center;
-    flex-direction: column;
+.section-3 {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
 }
 
-#content img{
-    width: 45%;
-    height: 45%;
+#content img {
+  width: 45%;
+  height: 45%;
 }
 
-#content hr{
-    width: 23%;
-    height: 4px;
-    margin-top: 0;
-    background-color: #00B172;
+#content hr {
+  width: 23%;
+  height: 4px;
+  margin-top: 0;
+  background-color: #00b172;
 }
 
-.container h3{
-    font-weight: bold;
-    color :#00B172;
-    font-size: 45px;
+.container h3 {
+  font-weight: bold;
+  color: #00b172;
+  font-size: 45px;
 }
 
-.container p{
-    color: #1a1a1a;
-    font-size: 30px;
-    text-align: center;
+.container p {
+  color: #1a1a1a;
+  font-size: 30px;
+  text-align: center;
 }
 
-.discover{
-    background-color: #00B172;
-    background-size: cover;
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    width: 100%;
+.discover {
+  background-color: #00b172;
+  background-size: cover;
+  display: flex;
+  flex-direction: row;
```

Comment:
Well done Fareaha!!

May be a little bit of margin right to your nav links would have made them more readable. Keep it up!